tests/internal/cluster/config.go

```diff
@@ -15,6 +16,77 @@ import (
 	configv1 "github.com/openshift/api/config/v1"
 )
 
+// GetOCPVersion retrieves the OCP clusterversion object from an arbitrary cluster and get current ocp version value.
+func GetOCPVersion(apiClient *clients.Settings) (string, error) {
+	glog.V(90).Infof("Gathering OCP version from cluster at %s", apiClient.KubeconfigPath)
+
+	clusterVersionObj, err := GetOCPClusterVersion(apiClient)
+	if err != nil {
+		return "", err
+	}
+
+	ocpVersion := clusterVersionObj.Object.Status.Desired.Version
+
+	return ocpVersion, nil
+}
+
+// CompareOCPVersionWithCurrent compares current OCP versions with the provided value.
+func CompareOCPVersionWithCurrent(apiClient *clients.Settings,
+	referenceOCPVersion string,
+	isGreater, orEqual bool) (bool, error) {
+	if apiClient == nil {
+		return false, fmt.Errorf("'apiClient' cannot be empty")
+	}
+
+	if referenceOCPVersion == "" {
+		return false, fmt.Errorf("'referenceOCPVersion' cannot be empty")
+	}
+
+	currentOCPVersion, err := GetOCPVersion(apiClient)
+
+	if err != nil {
+		return false, err
+	}
+
+	glog.V(100).Infof("The apiClient is empty")
```

Comment:
```suggestion
	glog.V(90).Infof("The apiClient is empty")
```
Level 100 is reserved for eco-goinfra

Reply:
Fixed- thanks.